tutorials/npm-Enterprise-on-a-closed-network.md

```diff
@@ -0,0 +1,62 @@
+# Npm Enterprise on closed network
+
+You can use npm Enterprise on the private/closed network. Here are the two approaches:
+
+1. Install npmE in open network and move VM’s to closed network
+2. Air-Gap installation
+
+### Install npmE in open network and move VM’s to closed network
+
+To achieve this create two VM’s let’s say npmBlank and npmSource on open-network and install npm Enterprise on both these VM’s.
+
+Both these VM’s originate as npm Enterprise servers, provisioned using the instructions provided [up and running section]( https://npme.npmjs.com/docs/up-and-running/platforms/other.html)
+
+Now, follow the instructions below.
+
+* Seed the VM npmSource (currently connected to the Internet) with the packages you need.
+* For seeding packages to this VM you can use White-listing or mirror policy.
+* Bring this VM (npmSource) to the closed network.
+* Also bring npmBlank that has no packages on it to the closed network. (It’s just a blank installation of npm Enterprise).
+* Now set up the blank npm Enterprise (i.e npmBlank) to mirror the npmSource VM with your packages on it.
```

Comment:
Please change this block to:

To initialize your installation:

 1. With `npmeSource` still connected to the internet, edit the `whitelist` file so that it's a new-line delimited list of the package names you'll want available on your closed network. The `white-list` file is in the data folder which is `/usr/local/lib/npme/data` by default unless you changed this under `Miscellaneous Data` in the npme Settings Panel.
 1. Allow adequate time for the replication process to pull in all the packages added to the `whitelist` file.
 1. Bring `npmeSource` and `npmePrimary` into the closed network.
 1. In `npmePrimary`'s Configuration Settings, under `Upstream Registry`:
    1. point the Upstream URL to `npmeSource`
    1. set the `Upstream Secret` to match `npmeSource`'s
    1. set the `Policy` to `mirror`